caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -238,14 +235,6 @@ div.tablePopover:hover {
 .ace_content {
     background-color: #f4f4f4;
 }
-.ws-el > .ws-el-controls {
```

Comment:
I took the time to do this because I thought it was better, let's talk and agree before ripping it out.